src/main/java/uk/gov/digital/ho/hocs/client/casework/dto/CreateMigrationCaseRequest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.digital.ho.hocs.client.casework.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import uk.gov.digital.ho.hocs.client.workflow.dto.DocumentSummary;
+
+import javax.validation.constraints.NotNull;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@EqualsAndHashCode
+public class CreateMigrationCaseRequest {
+
+    @JsonProperty("type")
+    private String type;
+
+    @JsonProperty("dateReceived")
+    private LocalDate dateReceived;
+
+    @JsonProperty("documents")
+    private List<DocumentSummary> documents;
+
+    @JsonProperty("data")
+    private Map<String, String> data;
+
+    @NotNull
```

Comment:
Is there actually value to this? In my opinion `type` would also be NotNull.

Reply:
Removed.